src/XRP/serializer.ts

```diff
@@ -48,27 +48,10 @@ const serializer = {
   ): TransactionJSON | undefined {
     const object: TransactionJSON = {
       Account: '',
+      Fee: '',
       Sequence: 0,
-      SigningPubKey: '',
       LastLedgerSequence: 0,
-      Fee: '',
-    }
-
-    const sequence = transaction.getSequence()?.getValue()
-    if (sequence) {
-      object.Sequence = sequence
-    }
-
-    const signingPubKeyBytes = transaction
-      .getSigningPublicKey()
-      ?.getValue_asU8()
-    if (signingPubKeyBytes) {
-      object.SigningPubKey = Utils.toHex(signingPubKeyBytes)
-    }
-
-    const lastLedgerSequence = transaction.getLastLedgerSequence()?.getValue()
-    if (lastLedgerSequence) {
-      object.LastLedgerSequence = lastLedgerSequence
+      SigningPubKey: '',
```

Comment:
Same order as the `interface BaseTransactionJSON` above. Not sure if there's a canonical order here.